### ISSUE TITLE
[Supporting the platform -Query Pattern]: 'Select a data source' Combo box disappears after resizing the page at 200%.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66580]
## User Experience: 
Users who rely on AT will face difficulty and can get confused if at 200% zooming, the controls are not visible properly or overlapped or get disappeared, then the user will not be able to access complete information present on the page. Note: User credentials should NOT be included in the bug.

## Repro Steps:
Open #Url: https://play.grafana.org/. Sign in with valid credentials. "Home" page appears.
Navigate to the left navigation bar and to the "Explore" button, select it. New "Explore" page appears, Navigate it.   Navigate to the "Query pane's first row and expand it. Navigate to the "Query Patterns" combo box.  Verify and observe the issue.

## Actual Result: Issue: 
'Select a data source' Combo box disappears after resizing the page at 200%.
## Expected Result:
Solution: "Select a data source" combo box should be present and clearly visible in their respective position. Also, any control should not get disappeared.

**Comment by ifrost** [comment]
> Navigate to the "Query Patterns" combo box. Verify and observe the issue.

We don't have "Query Patterns" on Explore page, could this be part of a query editor? If yes, which one was used in the test?

**Comment by Elfo404** [comment]
@grafana/observability-logs i suspect this has something to do with the loki query editor. feel free to bounce this back if not

**Comment by Elfo404** [comment]
read this again and the issue has nothing to do with the reproduction steps, i *think* this was fixed by https://github.com/grafana/grafana/pull/65149
